utils/dependency_cross_matcher.py

```diff
@@ -61,7 +61,7 @@ def main(argv=[], prog_name=''):
     exceptions = {
         'base64': ['v0.13.1', 'v0.21.4'],
         'bindgen': ['v0.57.0', 'v0.66.1'],
-        'bitflags': ['v1.3.2', 'v2.4.0'],
```

Comment:
I wanted to print the exceptions so that we are always aware of those and see if there was any room for improvement, otherwise they will just build up.
I can add a print for the diff between the exceptions and the mismatches if that's better?

Reply:
I ended up printing the exception list and the mismatcher list. It shouldn't be too difficult to figure out the problem if there is one.